### PR TITLE
translated readme to two more languages

### PR DESCRIPTION
**Pull request by pmayur**
translated readme to two new languages:

- hindi
- marathi

### CODE REVIEW

**Comment by SubhrajyotiSen** [comment]
@pmayur Can you please fix the merge conflict? 😃 
@aaossa I can review the Hindi translation. I will try finding someone to verify the marathi one soon

File: human.txt [hunk 1]
```diff
@@ -37,6 +37,7 @@ muan
 nik849
 petgoldfish
 pipmonkey
+pmayur
```
Comment (SubhrajyotiSen) [inline human.txt]: This has already been added in #88

**Review by SubhrajyotiSen — CHANGES_REQUESTED** [review]

**Comment by aaossa** [comment]
Thanks @SubhrajyotiSen :ok_hand: I don't know those languages, but I knew that you were a good reviewer for this :raised_hands:

File: human.txt (hunk 1 above)
Comment (pmayur) [reply human.txt]: I think my pmayur.html which happens to be my html file is conflicting where I made some changes intentionally. If thats not the problem can you help me with resolving the conflict?

**Comment by aaossa** [comment]
I fixed the conflicts, but will wait until @SubhrajyotiSen  say everything is OK 👍

**Comment by SubhrajyotiSen** [comment]
The Hindi translation is good. I am still looking for a person to review the marathi one.
@pmayur Just a suggestion. In the future, try to keep such commits in differrent PRs. For example, if you had made two separate PRs for Hindi and Marathi, I could have merged the Hindi one right now. But in the current case, the PR will have to wait till Marathi is reviwed.

**Comment by pmayur** [comment]
Well yeah thankyou ! these are the very first of my contributions so lesson learnt, thanks for the suggestion.

**Comment by aaossa** [comment]
Any updates on this?

**Comment by SubhrajyotiSen** [comment]
Unfortunately, I was not able to find anyone to verify the marathi translation

**Comment by aaossa** [comment]
Well, I think we could trust. That's the first step in OSS development isn't it? I'll give my 👍

**Review by aaossa — APPROVED** [review]
LGTM :sparkles:

**Review by SubhrajyotiSen — APPROVED** [review]

**Comment by SubhrajyotiSen** [comment]
@pmayur Thanks for the translations 😃

**Comment by pmayur** [comment]
thanks to you guys too  !

**Pull request merged**